nuitka/nodes/BuiltinOperationNodeBasesGenerated.py

```diff
@@ -1,20 +1,3 @@
-#     Copyright 2022, Kay Hayen, mailto:kay.hayen@gmail.com
```

Comment:
That is actually a weakness of the generation code that bitten me. I only run it where it gets later added, not good for collaboration. It should instead add them for you.